ospo-mindmap/Content/ospomindmap.md

```diff
@@ -58,9 +58,23 @@ development team of the Linux Project
 
 ## 🧩 Responsibilities
 
-### OS Legal Education
+### 📘 Develop and Execute Open Source Strategy
 
-#### Manage Legal Risks and Licenses
+- Open Source community, contributions and leadership
+- Compliance
+- InnerSource
+- Managing inquiries from customers, employees, investors or community members.
+- Risk tolerance
+- M&A, corporate development
+- Software procurement
+- Marketing
+
+### 🔍 Oversee Open Source Compliance
+
+- Comply with open source license obligations
+- Faciliate effective use of open source in proprietary products or services
+- Comply with third-party software supplier or customer contractural obligations
+- Protect company's intellectual property and product differentiation
```

Comment:
Suggestion: let's split Protect company's intellectual property and product differentiation into two:

- Protect company's intellectual property
- Protect company's product differentiation

And add under "Protect company's intellectual property": 

- "Open Source Due Diligence for acquisitions (or M&A projects)"

Reply:
@winterrocks Like your changes and added some things on top. FYI I am using organization instead of company in the doc as OSPO also exist outside of companies e.g. universities, non-profits, governments, etc.

```
- Protect organization's intellectual property
  - Conduct open source due diligence for acquisitions (or M&A projects)
  - Check new organization's open source project name does not infringe third party trademarks
- Protect organization's product differentiation
  - Verify no trade secrets are included in code which is to be open sourced
 ```